plugins/utils/harpoon.nix

```diff
@@ -27,7 +27,7 @@ in {
 
       package = helpers.mkPackageOption "harpoon" pkgs.vimPlugins.harpoon;
 
-      enableTelescope = mkEnableOption "Enable telescope integration";
+      enableTelescope = mkEnableOption "enable telescope integration";
```

Comment:
```suggestion
      enableTelescope = mkEnableOption "telescope integration";
```

Reply:
`mkEnableOption` already prefixes your provided description with `Whether to enable `.